app/views/play/level/tome/TomeView.coffee

```diff
@@ -118,8 +118,15 @@ module.exports = class TomeView extends CocoView
     else
       classroomCodeFormatDefault = @options.classroomAceConfig?.codeFormatDefault or 'text-code'
       classroomCodeFormats = @options.classroomAceConfig?.codeFormats or ['blocks-icons', 'blocks-text', 'blocks-and-code', 'text-code']
-      desiredCodeFormat = me.get('aceConfig')?.codeFormat or classroomCodeFormatDefault
+      desiredCodeFormat = me.get('aceConfig')?.codeFormat
+      if @options.level.get('product') == 'codecombat-junior'
```

Comment:
do we need these changes with query variable below now?

Reply:
these changes still work for new users. query variable now only work for teacher-dashboard. i think we can keep them until we make new logic for codeformat